### ISSUE TITLE
Forced to logout with horse

### ISSUE THREAD

**Issue by GDMush** [ImpactDevelopment/ImpactClient#1050]
### What happens?
TLDR; Entityspeed on horse causes me to glitch and force to log out to fix
Every so often (i play 2b2t with impact). I'll be on a horse going fast with entity speed down the highway (which is directly straight in a direction). I'll start glitching out and wont be able to move, my screen starts to go back and forth. I can go into freecam and i can see that my character is going back and forth really fast, sitting down like on a horse and is invisible. I cant break anything and I can't jump or shift to get off the horse. The only thing thats been fixing it is relogging. the problem is 2b2t's queue (i plan on getting pror queue in the future). I don't know if this glitch happens because of impact. Or because of 2b2t's low TPS sometimes. I think this might be due to be going faster then the chunks loading. Also when I reconnect I'm back on the horse. EDIT: This happened on Impact 4.5 1.12 and 4.6 1.12

### Exact steps to reproduce?

- Step one Horse with Entity Speed 9. Server fluctuating TPS 5-20

- Another step Straight direction flat land. 
  - Don't use auto walk or baritone. Just go foward with the horse using entityspeed 9
- I don't know why this happens, I have no idea if its easy to reproduce, This happens on 2b2t though.

<details> 
  <summary><strong>Game output</strong></summary>
 

```
I'm glitching back and forth. There is no output from the game, i'm just forced to logout to fix it. Also if I go in freecam the next chunks infront of my glitching character don't load.
```

</details>

<details> 
  <summary><strong>Crash report</strong></summary>
 

```
no crash just unable to interact with anything.
```

</details>

<details> 
  <summary><strong>Additional Client Info</strong></summary>
 

- Impact Version: "4.5 and 4.6 for 1.12.2" 
- Using Optifine? "No"
- Using LiteLoader? "No"
- Enabled mods: "Riding speed 9, ESP"
- Operating System: "Windows 10 64bit" 

</details>

**Comment by GDMush** [comment]
Nope. I'm not able to rn, because i'm in school. I can test when I get home though. I'm relating it to 2b2t because of the unstable TPS and lag. I'm guessing It may be me going faster or too fast then the lag/tps because the next chunk will not load. Also the horse disappears until I relog.

**Comment by 0-x-2-2** [comment]
this is a server thing
